app/src/main/java/com/antosik/benchproject/app/common/binding/swipeRefreshLayout/SwipeRefreshLayoutBindingAdapters.kt

```diff
@@ -0,0 +1,12 @@
+package com.antosik.benchproject.app.common.binding.swipeRefreshLayout
+
+import androidx.databinding.BindingAdapter
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+import com.antosik.benchproject.app.common.state.UIState
+
+@BindingAdapter("hideIfRefreshed")
+fun SwipeRefreshLayout.hideIfRefreshed(uiState: UIState) {
```

Comment:
I created it to omit every showing swipe layout progress bar when I load data.

Reply:
Avoid `if` in names, especially in binding adapters. Here you could use just `isRefreshing`. But probably `isRefreshing` is already defined (could be used without your custom binding adapter). So you could create some function on `UIState` `isRefreshing` and use it like that `app:isRefreshing:"{uiState.isRefreshing()}"`. This way you don't have logic in binding adapter (you should avoid logic in binding adapters).

Reply:
I created an appropriate method for this and I changed the name which contains `if` in the title.